HelloWorld/Main.java

```diff
@@ -1,5 +1,9 @@
-int target = -5;
-int num = 3;
+package HelloWorld;
 
-target = -nua;  
-target += num;
+public class Main
+{
+    public static void main(String[] args)
+    {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.alumni-services-001.com/coding_rules#rule_key=squid%3AS106)